compiler/natives/src/syscall/js/js.go

```diff
@@ -281,3 +281,27 @@ func (e *ValueError) Error() string {
 type Wrapper interface {
 	JSValue() Value
 }
+
+// CopyBytesToGo copies bytes from the Uint8Array src to dst.
+// It returns the number of bytes copied, which will be the minimum of the lengths of src and dst.
+// CopyBytesToGo panics if src is not an Uint8Array.
+func CopyBytesToGo(dst []byte, src Value) int {
+	vlen := src.v.Length()
+	if dlen := len(dst); dlen < vlen {
+		vlen = dlen
+	}
+	copy(dst, src.v.Interface().([]byte))
```

Comment:
Could you add a type assertion in the `src` here to be consistent with `CopyBytesToJS`?